src/physics/em/detail/BetheHeitlerInteractor.i.hh

```diff
@@ -149,34 +150,20 @@ CELER_FUNCTION Interaction BetheHeitlerInteractor::operator()(Engine& rng)
     real_type                          phi = sample_phi(rng);
 
     // Electron
-    real_type cost = this->sample_cos_theta(secondaries[0].energy.value(), rng);
+    TsaiUrbanDistribution sample_electron_angle(
+        secondaries[0].energy, units::MevMass{shared_.inv_electron_mass});
```

Comment:
The units here are wrong but it looks like the quantity is right. The sampler and its G4ModifiedTsai counterpart both expect 1/(m_e c^2) so units of `MevMassInv`, which you can define (for now, and I'd dprob put it in the `TsaiUrbanDistribution`) as
```c++
using MevMassInv = Quantity<UnitDivide<CLightSq, Mev>>;
```